catalog/nsili/nsili-app/src/main/resources/features.xml

```diff
@@ -43,7 +43,7 @@
     </feature>
 
     <feature name="nsili-app" install="auto" version="${project.version}"
-             description="The Alliance NSILI App provides interoperability with STANAG 4559 compliant services. ::Alliance NSILI">
+             description="The Alliance NSILI App provides interoperability with STANAG 4559 compliant services.">
```

Comment:
Why this change?  I think `::Alliance NSILI` puts the name on the app in the Admin UI